examples/openfortivpn.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description = Forti VPN for default
+
+[Service]
+ExecStart = /usr/bin/openfortivpn --use-syslog -c /etc/openfortivpn/config
```

Comment:
Can you remove `--use-syslog`? It is not the default and most systems use Systemd's journal.